src/const/paths.ts

```diff
@@ -3,6 +3,7 @@
  */
 export default class AppPaths {
   static HOME = '/';
-
+  static SIGN_IN = '/signin';
   static CALC = '/calc';
```

Comment:
Move `CALC` to the very bottom because it's just a sample and unlikely to stay there